### ISSUE TITLE
Add the use of / in chat console. 

### ISSUE THREAD

**Issue by Deeptanil** [DiscordSRV/DiscordSRV#1269]
### Why?

Cuz y not, jk.
Because it's just convenient. 

### What and How?

Add use of / in chat Channel. 

### Are there alternatives?

No. 

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

### Anything else

Nothing else.

**Comment by BoredManCodes** [comment]
no.
issue closed.






jk

**Comment by Dinty1** [comment]
Do you mean having Minecraft commands as slash commands?

**Comment by Deeptanil** [comment]
I mean,
In the "console chat" in discord, we do not need to use "/". I'm suggesting that / and without / to be compatible in console chat channel.

**Comment by James-Bennett-295** [comment]
Why would that be useful?

**Comment by Deeptanil** [comment]
Convince... Also just sometimes when I forget that / isn't needed... I gotta retype or copy it.
So just convince.

**Comment by Scarsz** [comment]
While I'm not for silently ignoring leading slashes (some commands specifically need them) we could add a mode that more closely mirrors the behavior of most online consoles.

If a slash isn't used, it can just be interpreted as a `say` command. Otherwise, if there are slashes, run it as a command as normal.

**Comment by Deeptanil** [comment]
Or we could have both....
If console recived a message and doesn't have /, use that as a command.
If console does recive with /, run the command too.

This also makes WorldEdit commands easier, but that's not why I asked for it. It's pretty okay like this... But just convience..

**Comment by Vankka** [comment]
I would personally not make the messages not starting with `/` not work as commands, this is how the actual console works after all. Additionally I think using `/` might interfere with Discord slash commands. Although I'm not completely against adding this, will need some thought though

**Comment by Scarsz** [comment]
> Or we could have both....
> 
> If console recived a message and doesn't have /, use that as a command.
> 
> If console does recive with /, run the command too.
> 
> 
> 
> This also makes WorldEdit commands easier, but that's not why I asked for it. It's pretty okay like this... But just convience.. 

What you're suggesting with making world edit commands "easier" isn't true. Doing what you're suggesting would mean either

- running everything with and without a leading command, leading to duplicates
- requiring **three** slashes to run a WorldEdit command, i.e. `///command`

**Comment by Deeptanil** [comment]
Yes, I think I understand these complications, the more I think of it... Let it be as it is xD

**Comment by Vankka** [comment]
I have already been thinking of ways to utilize slash commands for running Minecraft commands, however the 100 + 100 (server + global) limit somewhat limits it in that you wouldn't be able to have all Minecraft commands as their slash commands.

**Comment by Deeptanil** [comment]
No no no I don't mean minecraft commands! That's gotta be the server ops brain xD
I'm talking about the custom commands... The "DiscordCannedResponses" 

Also a bug? 
![Screenshot_20210930-220807_Chrome](https://user-images.githubusercontent.com/86883566/135496517-92e8d3d4-8850-4d17-96aa-37af5ea7e647.png)

**Comment by Deeptanil** [comment]
Doesn't show the img (bottom of the screenshot)

**Comment by Vankka** [comment]
https://github.com/DiscordSRV/DiscordSRV/issues/384 / yeah canned responses will probably be converted entirely to slash commands or at minimum have the option to be slash commands in the future

The Spigot page is due to be completely redone at somepoint, but thanks for letting us know
